src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -368,10 +368,18 @@ std::string statisticsTreat
     out += TAG_ADD_COUNTER("logRequests", noOfLogRequests);
   }
 
+#if 0
+  //
+  // The valgrind test suite uses REST GET /version to check that the broker is alive
+  // This fact makes the statistics change and some working functests fail under valgrindTestSuite
+  // due to the 'extra' version-request in the statistics.
+  // Removing version-requests from the statistics 'report' fixes this problem.
+  //
   if (noOfVersionRequests != -1)
   {
     out += TAG_ADD_COUNTER("versionRequests", noOfVersionRequests);
   }
+#endif
```

Comment:
So, if In understand correctly we have two potencial solutions on the table a) using REGEX((1|2)), b) remove the code in the #if 0/#endif and we have to chose between one of them?

Maybe the less intrusive is the one based on REGEX((1|2)) in just one .test (i.e. the .test for statistics) explaining in a comment in that .test why the REGEX is needed.